salt/metalk8s/kubeadm/init/certs/installed.sls

```diff
@@ -0,0 +1,3 @@
+Install m2crypto:
```

Comment:
I would put this in a earlier stage of the bootstrap as a pre requisite, so that we could assume that we can use x509 module at any time, without requiring this state each time

Reply:
We could but btw it's better to include it everywhere so we can call directly any cert state

Reply:
We won't execute this state a lot of times, so let's keep it there.
Installing packages for later use is not part of the bootstrap step.